docs/writing-rules/data-flow/taint-mode.md

```diff
@@ -9,6 +9,9 @@ description: >-
 
 Semgrep supports intra-procedural [taint tracking](https://en.wikipedia.org/wiki/Taint_checking). This is a kind of data-flow analysis that tracks the flow of untrusted (aka "tainted") data throughout a program. The analysis produces a finding whenever such data goes into a vulnerable function (aka "sink"), without first having been checked or transformed accordingly (aka "sanitized").
 
+The following video can help you to understand taint mode:
+<iframe class="yt_embed" width="100%" height="432px" src="https://www.youtube.com/embed/6MxMhFPkZlU?start=25" frameborder="0" allowfullscreen></iframe>
+
```

Comment:
@efortuna I am tempted to move this to line 9 - so this video will become the actual introduction. We can let it start from the beginning also. :-)

Reply:
I think prooooobably start with the text. but yeah maaaaybe let the video start at the beginning as it gives a summary of what's to come in the video? not a strong preference though so I defer to you! starting at "25" jumps right to the goodies :-)

Reply:
I like the start! Let's put it from the start. :-)